testindex/validators.py

```diff
@@ -297,8 +297,26 @@ def run(self):
             # Else clone was success, check the git path
             git_path = clone_path + "/" + str(entry["git-path"])
 
+            # * Check for pre-build script
+            # TODO : Make a better implementation of pre-build script checking
+            # TODO : Ideally, if prebuild is not in entry, it wont reach here and this should happen if it is not None
+            prebuild_exists = False
+            if constants.PREBUILD_SCRIPT in entry and constants.PREBUILD_CONTEXT in entry:
+                prebuild_exists = True
+                prebuild_script = entry.get(constants.PREBUILD_SCRIPT)
+                prebuild_context = entry.get(constants.PREBUILD_CONTEXT)
+                if (prebuild_script and not path.exists(path.join(git_path, prebuild_script))
```

Comment:
That's a lot of `and` and `not` in there. Can we have a short comment about what exactly we're doing here?

Reply:
@mohammedzee1000 : Why didn't this being addressed? Shouldn't a complex operation be decorated with comment(s) ?

Reply:
@bamachrn : I would like to point out here. If we see a reviewer has asked for a change, we must ensure it is addressed. You merging it without review comments addressed, makes no point of a multiple reviewers and their reviews.